lib/dealership.rb

```diff
@@ -0,0 +1,32 @@
+class Dealership
+  attr_reader :inventory,
+              :inventory_count
```

Comment:
You will want to have 
```
:name,
:location,
:inventory
```
in your `attr_reader`.  You will want to remove the `inventory_count` from your `attr_reader`.